cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -1443,6 +1458,11 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
+	/// Returns the claim queue offset.
+	pub fn fetch_claim_queue_offset() -> u8 {
```

Comment:
```suggestion
	pub fn get_claim_queue_offset() -> u8 {
```

Reply:
In rust you don't use the `get` prefix, so just `claim_queue_offset`